modules/end-user-guide/partials/proc_setting-up-che-editor-yaml.adoc

```diff
@@ -0,0 +1,23 @@
+[id="setting-up-che-editor-yaml_{context}"]
+= Setting up che-editor.yaml
+
+To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, you can use the `che-editor.yaml` file. This is convenient when, as a developer using {prod-short}, you clone your personal fork of the project repository into your {prod-short} user's workspaces, and this way you don't have to specify your IDE every time you start another workspace with a clone of that repository.
```

Comment:
@l0rd, great, thank you for correcting me. WDYT? 
```suggestion
To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, you can use the `che-editor.yaml` file. Using this file allows you to set a common default editor for your team and to provide new contributors with a default editor that is most suitable for your project.
```

Reply:
👍